app/src/main/java/rick_and_morty/data/model/InfoDto.kt

```diff
@@ -0,0 +1,17 @@
+package rick_and_morty.data.model
+
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class InfoDto(
+    @Json(name = "count")
+    val count: Int? = null,
```

Comment:
I think you rethink the nullable values in ALL DTOs. Is the API always delivering data, or is it possible that there might nullables be really needed no info available? This could improve further processing.

Reply:
Done